src/workerTemplate.js

```diff
@@ -22,14 +27,18 @@ let globalResultQueue;
  * @param {String} workerQueue The name of the worker queue to look for jobs
  * @param {String} resultQueue The name of the result queue to report back to
  * @param {Function} callBack The callback function getting called when a job is received
+ * @param {Function} areDependenciesAvailable Function to check if depencies of a worker are available
```

Comment:
could be renamed to e.g. `preconditionsCheck`

Reply:
good point. I renamed it